torch/_prims/__init__.py

```diff
@@ -1130,6 +1134,30 @@ def _remainder_nvfuser(fd: Any, a: TensorLikeType, b: TensorLikeType):
 #
 # View operations
 #
+atleast_1d = _make_elementwise_unary_prim(
```

Comment:
I don't think these should be prims, rather, they should be expressed through unsqueeze?